.husky/pre-commit

```diff
@@ -1,9 +1,10 @@
 #!/usr/bin/env sh
 . "$(dirname -- "$0")/_/husky.sh"
 
-npm run format:fix
-npm run lint:fix
+npm run lint-staged #runs the lint checks for staged files only, check .lintstagedrc.json for config
+# npm run format:fix
```

Comment:
1. Why was this done?
2. Are the references to `format:fix` and `lint:fix` still being used from the code base or documented? If not they should be removed.

Reply:
the 'lint-staged' script is using these commands indirectly (check .lintstagedrc.json). I had to remove them only from the git hooks so that only staged files are checked for linting.